1.7/administration/upgrading.md

```diff
@@ -0,0 +1,157 @@
+---
+post_title: Upgrading
+---
+
+## Summary
+
+#### These steps provide instructions for upgrading a DC/OS cluster from DC/OS 1.6 to 1.7.  If performed on a supported OS with all prerequisites fulfilled, this upgrade _should_ preserve the state of running tasks on the cluster.  This guide follows some of the steps outlined in the Advanced DC/OS Installation Guide:
+https://dcos.io/docs/1.7/administration/installing/custom/advanced/
+
+#### The Advanced Installation method is the only recommended upgrade path for DC/OS.  Please familiarize yourself with the Advanced DC/OS Installation Guide before proceeding.
+
+## Prerequisites
+
+- Mesos, Mesos Frameworks, Marathon, Docker and all running tasks in the cluster should be stable and in a known healthy state.
+-  You must have access to copies of the config files used with DCOS 1.6: `config.yaml` and `ip-detect`
+- You must be using systemd 218 or newer to maintain task state
+- Be sure that all hosts (masters and agents) can communicate with all other hosts on port `53|UDP`
+- In CentOS|RedHat, `$ sudo yum install -y ipset` (used in some IP detect scripts)
+- You must be familiar with using `systemctl` and `journalctl` command line tools to review and monitor service status. Troubleshooting notes can be found at the end of this document.
+
+## Instructions
+
+These upgrade instructions follow the general procedural patterns of the Advanced DC/OS Installation Guide.  Please familiarize yourself with the Advanced DC/OS Installation Guide before performing an upgrade of DC/OS:
+https://dcos.io/docs/1.7/administration/installing/custom/advanced/
+
+### Bootstrap Node(s)
+
+#### Copy and Update config.yaml and ip-detect
+
+Copy the DC/OS 1.6 config.yaml and ip-detect script to a new, clean folder on your bootstrap node. 
+
+**IT IS NOT POSSIBLE TO CHANGE THE `exhibitor_zk_backend` SETTING DURING AN UPGRADE**
+
+Please note, the syntax of the DC/OS 1.7 config.yaml differs from that of DC/OS 1.6.  For a detailed description of config.yaml syntax and parameters, see:
+https://dcos.io/docs/1.7/administration/installing/custom/configuration-parameters/
+
+#### Continue Bootstrap Setup
+
+Once you have merged your config.yaml into the config.yaml for the new version, proceed to build your installer package:
+
+- Download the file dcos_generate_config.sh.
+- Generate the installation files.
+- Disable Docker restarts in `dcos_install.sh` *This step is critical to prevent task restarts*
+
+```
+sed -i -e "s/systemctl restart systemd-journald//g" -e "s/systemctl restart docker//g" genconf/serve/dcos_install.sh
+```
+
+- Run the nginx container to serve the installation files.
```

Comment:
It would help to either add the instruction here, or link to it:
https://github.com/dcos/dcos-docs/blame/master/1.7/administration/installing/custom/advanced.md#L159

It should be possible to add a direct link by adding `<a name="nginx-run-command"></a>` into the install doc, and linking it via `/docs/1.7/administration/installing/custom/advanced#nginx-run-command`